README.rst

```diff
@@ -52,7 +45,23 @@ This is optional. You can set your own return url by adding this to `settings.py
 
     PESAPAL_OAUTH_CALLBACK_URL = 'app_name:url_name'  # this needs to be a reversible
 
-#. Run `python manage.py syncdb` to create the polls models.
+The url can have the 'merchant_reference' passed to it as a parameter for further processing.
+Or be without a parameter.
+
+for example the url pattern
+
+r'^response/$'
+
+without the merchant reference parameter
+
+or
+
+r'^response/(?P<merchant_reference>\d+)/$'
```

Comment:
I used an integer since the existing merchant_reference field in the model is defined as an integer field, but adding support for a string should be trivial.